src/components/WhoWeAre/WhoWeAre.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { background, container, h1, p } from '../WhoWeAre/WhoWeAre.module.scss';
+
+export const WhoWeAre = () => {
+  return (
+    <section className={`${container}`}>
+      <div className={`${background}`}>
+        <h1 className={`${h1}`}>Kim jesteśmy?</h1>
+        <p className={`${p}`}>
```

Comment:
ten blok nei ejst przez to że blok-rodzic ma 140% rozłożony na 100% rzerokości (szczególnie od lewej krawędzi), więc jak dodam ten sam tekst co juz jest 2 raz (po prostu przekopiuję i wkleję by był 2 razy dłuższy) to powstaje nieprzyjemny efekt, że blok zakrzywiony jest `pusty` na dole:
![image](https://github.com/UEK-i-team/team-homepage/assets/49104061/9bd4b524-b3a5-44c5-9654-6f913f4dbc5d)

Reply:
Dodatkowo oczywiście generuje to niepotrzebnego scrolla, bo zamiast najpierw wypełnić blok tekstem i potem dokłądac scrolla, to on od razu dokłada scrolla a miejsce i tak zostaje przez padding